src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -241,6 +241,14 @@
       "Reduce healing by %1$s%%."
     ]
   },
+  "stsjorbsmod:ShrapnelBloomPower": {
+    "NAME": "Shrapnel Bloom",
+    "DESCRIPTIONS": [
+      "This turn, your next",
```

Comment:
I don't think mixing the stacking behavior does quite the right thing; if you somehow have a Shrapnel Bloom and a Shrapnel Bloom+ and play both, I think what's currently going to happen is the number of card will stack but the number of plays per card will use the first value and ignore the second. Asked jorbs to confirm the desired stacking behavior in discord.

Reply:
Per discord, let's go with the same thing the Wanderer's `Gather Power` does, where "stacking increases the number of times the next one card is played, it never affects multiple next cards". That probably suggests we should update the wording on the card/power to be more consistent with Gather Power and say `This turn, your next non-Legendary card is played !M! additional time.` rather than `...played 2 times`, since it makes the stacking behavior more obvious to a player.